tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_provision.c

```diff
@@ -130,6 +298,60 @@ static void test_provisioner_pb_adv_no_oob(void)
 	PASS();
 }
 
+static void test_device_pb_adv_oob_auth(void)
+{
+	int err = 0;
+
+	k_sem_init(&prov_sem, 0, 1);
+
+	oob_auth_set();
+
+	oob_channel_id = bs_open_back_channel(0, (unsigned[]){1}, (unsigned[]){0}, 1);
```

Comment:
A sidecomment: Maybe comment in the code here (and in the test script) that this test assumes to be running in device 0, and expects the other side of the test in device 1, and viceversa in the other test.